maven-plugin/src/main/java/org/openmrs/maven/plugins/utility/ModuleInstaller.java

```diff
@@ -24,98 +10,113 @@
-        }
-        installModules(coreModules, server.getServerDirectory().getPath());
-    }
-
-    public void installModulesForDistro(Server server, DistroProperties properties, DistroHelper distroHelper) throws MojoExecutionException {
-        List<Artifact> coreModules;
-        // install other modules
-        coreModules = properties.getWarArtifacts(distroHelper, server.getServerDirectory());
-        if (coreModules == null) {
-            throw new MojoExecutionException(String.format("Invalid version: '%s'", server.getVersion()));
-        }
-        installModules(coreModules, server.getServerDirectory().getPath());
-        File modules = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_MODULES);
-        modules.mkdirs();
-        List<Artifact> artifacts = properties.getModuleArtifacts(distroHelper, server.getServerDirectory());
-        // install modules for each version
-        installModules(artifacts, modules.getPath());
-    }
-
-    public void installModules(List<Artifact> artifacts, String outputDir) throws MojoExecutionException {
-        final String goal = "copy";
-        prepareModules(artifacts.toArray(new Artifact[0]), outputDir, goal);
-    }
-
-    public void installModules(Artifact[] artifacts, String outputDir) throws MojoExecutionException {
-        final String goal = "copy";
-        prepareModules(artifacts, outputDir, goal);
-    }
-
-    public void installModule(Artifact artifact, String outputDir) throws MojoExecutionException {
-        final String goal = "copy";
-        prepareModules(new Artifact[] { artifact }, outputDir, goal);
-    }
-
-    /**
-     * Handle list of modules
-     * @param artifacts
-     * @param outputDir
-     * @param goal
-     * @throws MojoExecutionException
-     */
-    private void prepareModules(Artifact[] artifacts, String outputDir, String goal) throws MojoExecutionException {
-        MojoExecutor.Element[] artifactItems = new MojoExecutor.Element[artifacts.length];
-        for (int index = 0; index < artifacts.length; index++) {
-            artifactItems[index] = artifacts[index].toElement(outputDir);
-        }
-
-        List<MojoExecutor.Element> configuration = new ArrayList<>();
-        configuration.add(element("artifactItems", artifactItems));
-        if (goal.equals(GOAL_UNPACK)) {
-            configuration.add(element("overWriteSnapshots", "true"));
-            configuration.add(element("overWriteReleases", "true"));
-        }
-        executeMojo(
-                plugin(
-                        groupId(SDKConstants.DEPENDENCY_PLUGIN_GROUP_ID),
-                        artifactId(SDKConstants.DEPENDENCY_PLUGIN_ARTIFACT_ID),
-                        version(SDKConstants.DEPENDENCY_PLUGIN_VERSION)
-                ),
-                goal(goal),
-                configuration(configuration.toArray(new Element[0])),
-                executionEnvironment(mavenProject, mavenSession, pluginManager)
-        );
-    }
+	
```

Comment:
I don't love all the reformatting in this PR. It makes it very hard to review and know if anything has changed and if so, what.

Reply:
Please don't commit files you haven't touched other than reformatting. If there are any changes here, can you please use GitHub comments to indicate what those are?